GCRCatalogs/dc2_object.py

```diff
@@ -171,8 +196,15 @@ def __init__(self, file_handle, key, schema=None):
         super(ObjectTableWrapper, self).__init__(file_handle, key, schema)
         # Add the schema info for tract, path
         # These values will be read by `get_constant_array`
-        self._schema['tract'] = {'dtype': int, 'default': self.tract}
-        self._schema['patch'] = {'dtype': str, 'default': self.patch}
+        self._schema['tract'] = {'dtype': 'int64', 'default': self.tract}
+        self._schema['patch'] = {'dtype': '<U3', 'default': self.patch}
```

Comment:
In practice, this will be fine, but formally patches could be indexed by a more than one-digit number.  E.g., patch `31,5`

If this happened, it would break some other stuff where we do int<->string conversions for patch names.  But perhaps throw in a comment to the effect that we are assuming single-digit patch indexes.

Reply:
Good point. There's in fact no need to specify the number of character for this particular case. 

Code updated. Great if you can re-approve. Thanks!